plugins/config/commands

```diff
@@ -90,26 +118,8 @@ case "$1" in
     ;;
 
   config:set)
-    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
-    verify_app_name "$2"
-    APP="$2"
-
-    if [[ -z "${*:3}" ]]; then
-      echo "Usage: dokku config:set APP KEY1=VALUE1 [KEY2=VALUE2 ...]"
-      echo "Must specify KEY and VALUE to set."
-      exit 1
-    fi
-
-    shift 2
-    dokku config:set-norestart $APP "$@"
-    dokku_log_info1 "Restarting app $APP"
-    dokku ps:restart $APP
-   ;;
-
-  config:set-norestart)
-    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
-    verify_app_name "$2"
-    APP="$2"
+    parse_config_args "$@"
+    [[ "$2" = "--no-restart" ]] && set -- "${@:1:1}" "${@:3}"
```

Comment:
couldn't get around doing this again as `parse_config_args` is run in subshell, unless we have two functions (one to set env vars and one to echo new `$@` and use it. meh...